### PR TITLE
Rebase to 1.21.0

### PR DESCRIPTION
**Pull request by jsafrane**
/kind cleanup
**What this PR does / why we need it**:
Update to the latest release-tools and rebase all Kubernetes libs to v1.21.0

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,55 +13,51 @@ require (
...
-	k8s.io/kubectl => k8s.io/kubectl v0.20.0
-	k8s.io/kubelet => k8s.io/kubelet v0.20.0
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.0
-	k8s.io/metrics => k8s.io/metrics v0.20.0
-	k8s.io/mount-utils => k8s.io/mount-utils v0.20.1-rc.1
-	k8s.io/node-api => k8s.io/node-api v0.19.0
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.0
-	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.0
-	k8s.io/sample-controller => k8s.io/sample-controller v0.20.0
+	k8s.io/api => k8s.io/api v0.21.0
```
Comment (msau42) [inline go.mod:36]: I believe all of these replacement statements are only needed if the repo is importing "k8s.io/kubernetes"

Comment (jsafrane) [reply go.mod:36]: Filed https://github.com/kubernetes-csi/csi-release-tools/pull/140 to update `go-get-kubernetes.sh` that's used to rebase Kubernetes libs.

Comment (jsafrane) [reply go.mod:36]: Fixed with the new release-tools.

**Comment by Jiawei0227** [comment]
/cc @msau42 
for approval.

**Pull request merged**